### ISSUE TITLE
Export default function vs class

### ISSUE THREAD

**Issue by oligriffiths** [chadhietala/broccoli-rollup#47]
Hey folks

The direction we are looking to take plugins for Broccoli is that the default export be a function rather than the class definition, such that composing a broccoli pipeline looks like:
```js
import rollup from 'broccoli-rollup';

const js = rollup('app', {
  rollup: {
    input: 'app/index.js',
    output: {
      file: 'app.js',
      format: 'es',
    },
  }
});
```
I notice that https://github.com/chadhietala/broccoli-rollup/issues/9 was previously closed, but it looks like @stefanpenner previously made this suggestion.

See https://github.com/broccolijs/broccolijs.github.io/pull/8
https://broccoli.build/plugins.html#example-plugin

**Comment by krisselden** [comment]
@oligriffiths `const rollup = require('broccoli-rollup').default;` was a breaking change from `const rollup = require('broccoli-rollup');`  I'm happy to comply, it was just difficult to do this in a way that didn't break things.  I can likely just make a Proxy for modules.export for cjs that can deprecate the usage and proxy handler.apply to the default.

**Comment by oligriffiths** [comment]
@krisselden cool, whatever you can do to help would be great. I understand this would be a breaking change as it changes the default import semantics but this is something we are encouraging all plugin authors to adopt.

**Comment by krisselden** [comment]
should be fixed by https://github.com/chadhietala/broccoli-rollup/pull/51

**Comment by krisselden** [comment]
@oligriffiths it was a little unclear what the convention is for the exported class, reading it seems like it is pascal case of the node module name, that is what I went with.

**Comment by krisselden** [comment]
this is fixed as of version 4.1.0

**Comment by oligriffiths** [comment]
Thanks @krisselden 
The interface should basically match an ESM interface where you default export is a function and a named export of the class.
